x-pack/test/cases_api_integration/common/lib/api/index.ts

```diff
@@ -524,6 +524,7 @@ export const getCase = async ({
       `${getSpaceUrlPrefix(auth?.space)}${CASES_URL}/${caseId}?includeComments=${includeComments}`
     )
     .set('kbn-xsrf', 'true')
+    .set('x-elastic-internal-origin', 'foo')
```

Comment:
I added this in my tests where needed without thinking too much.

I assume this extra header doesn't break anything in existing tests?

Reply:
yes, I have verified this. It doesn't break existing tests.